### ISSUE TITLE
Pretrained robust backbone ResNet-50

### ISSUE THREAD

**Issue by YanghuiMo** [nmndeep/Robust-Segmentation#7]
Hi, thanks for sharing, its great work. Which Pretrained robust backbone ResNet-50 model did you used when PIR-AT training PSPNet? could you provide the download link?

**Comment by nmndeep** [comment]
Hi, thanks for your interest.
The pre-trained model is the Nr. 25 form the robustbench (ImageNet $\ell_\infty$) leaderboard, relating to this paper: [https://arxiv.org/abs/2007.08489](https://arxiv.org/abs/2007.08489)

**Comment by YanghuiMo** [comment]
Thanks for reply. What does this Nr. 25 mean? When I open https://github.com/RobustBench/robustbench/blob/master/robustbench/model_zoo/imagenet.py , which shows several robust ResNet 50 adversarially trained on imagenet. Then, I use the ('Salman2020Do_R50', {
            'model': lambda: normalize_model(pt_models.resnet50(), mu, sigma),
            'gdrive_id': '1TmT5oGa1UvVjM3d-XeSj_XmKqBNRUg8r',
            'preprocessing': 'Res256Crop224'

Not sure its the one you used？

<img width="602" height="338" alt="Image" src="https://github.com/user-attachments/assets/13f2b9f3-1873-40e4-9f4b-db0e3e070421" />

**Comment by nmndeep** [comment]
Hi, yes 'Salman2020Do_R50' should be the model that we use for PSPNet.

Hope this helps.
